operators/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN set -x \
     && chmod 755 /eck
 
 WORKDIR /eck
-USER elastic
+USER 101
```

Comment:
What's the difference?

Reply:
`runAsNonRoot` can only be checked on numeric id, no on the user name